.travis.yml

```diff
@@ -6,6 +6,8 @@ go:
     - 1.4
     - 1.5
     - 1.6
+    - 1.7
+    - latest
```

Comment:
Weird, I thought it was "latest", meant "tip". Should be working now


--
Kevin Burke
925.271.7005 | kev.inburke.com

On Tue, Aug 23, 2016 at 8:07 PM, Dmitri Shuralyov <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/satori/go.uuid/pull/37#discussion_r75988528>:
>
> > @@ -6,6 +6,8 @@ go:
> >      - 1.4
> >      - 1.5
> >      - 1.6
> > +    - 1.7
> > +    - latest
>
> Can you point out where this feature is documented? I am not able to find
> it on https://docs.travis-ci.com/user/languages/go page, I only see the
> following note there:
>
> Specifying a Go version to use
>
> You can use any tagged version of Go or use tip to get the latest version.
>
> Is latest documented somewhere else?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/satori/go.uuid/pull/37/files/698a53f804f9868c48eb8d1783f6ec4a830f4f2a#r75988528>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAOSI_oqQbg3IcUS1EyqDuBJbqy_KHHBks5qi7VggaJpZM4JrjJy>
> .
>